BitFlow/BitFlow.py

```diff
@@ -82,11 +82,8 @@ def __init__(self, model, dataset_size, size_w, size_output, data_range, true_wi
                     inputs["W"] = W
                     inputs["O"] = torch.Tensor(
                         1, size_output).fill_(true_width)[0]
-                    new_y = model(**inputs)
-                    if isinstance(new_y.tolist(), list):
-                        self.Y.append(new_y.tolist()[0])
-                    else:
-                        self.Y.append(new_y.tolist())
+                    new_y = torch.tensor(model(**inputs))
```

Comment:
Why the need to convert the tuple of outputs to a torch tensor?

Reply:
I was testing with it earlier, but it is no longer necessary so I have removed it — thanks!